FWCore/ParameterSet/python/Config.py

```diff
@@ -533,11 +533,17 @@ def extend(self,other,items=()):
                 if not item.hasLabel_() :
                     item.setLabel(name)
             elif isinstance(item,Schedule):
-                self.__setattr__(name,item)
+                if name == "schedule" and self.__schedule is not None:
+                    self.__schedule.extend(item)
+                else:
+                    self.__setattr__(name,item)
             elif isinstance(item,_Unlabelable):
                 self.add_(item)
             elif isinstance(item,ProcessModifier):
                 item.apply(self)
+            elif isinstance(item,Process):
```

Comment:
I don't think I like using ```extend``` and hence ```load``` in this way. A legitimate reason to have another ```cms.Processs``` is for ```SubProcess``` handling. Having a ```cms.Process``` pulled in via ```extend/load``` could break ```SubProcess```. I think keeping the seperate ```extendProcess``` method would be better.